scripts/completion/dvc.bash

```diff
@@ -4,63 +4,75 @@
+_dvc_fetch='-j --jobs -r --remote -a --all-branches -T --all-tags --all-commits -d --with-deps -R --recursive --run-cache'
 _dvc_fetch_COMPGEN=_dvc_compgen_DVCFiles
-_dvc_gc='-a --all-branches --all-commits -T --all-tags -w --workspace -c --cloud -r --remote -f --force -p --projects -j --jobs'
+_dvc_freeze=''
+_dvc_freeze_COMPGEN=_dvc_compgen_DVCFiles
+_dvc_gc='-w --workspace -a --all-branches -T --all-tags --all-commits -c --cloud -r --remote -f --force -j --jobs -p --projects'
 _dvc_get='-o --out --rev --show-url'
+_dvc_get_COMPGEN=_dvc_compgen_files
 _dvc_get_url=''
+_dvc_get_url_COMPGEN=_dvc_compgen_files
+_dvc_git_hook='pre-commit post-checkout pre-push'
+_dvc_git_hook_post_checkout=''
+_dvc_git_hook_pre_commit=''
+_dvc_git_hook_pre_push=''
 _dvc_import='-o --out --rev'
+_dvc_import_COMPGEN=_dvc_compgen_files
 _dvc_import_url='-f --file'
-_dvc_init='--no-scm -f --force'
-_dvc_install=''
+_dvc_import_url_COMPGEN=_dvc_compgen_files
+_dvc_init='--no-scm -f --force --subdir'
+_dvc_install='--use-pre-commit-tool'
 _dvc_list='-R --recursive --dvc-only --rev'
 _dvc_list_COMPGEN=_dvc_compgen_files
-_dvc_freeze=''
-_dvc_freeze_COMPGEN=_dvc_compgen_DVCFiles
-_dvc_metrics='add diff modify remove show'
-_dvc_metrics_add='-t --type -x --xpath'
+_dvc_metrics='add show diff remove'
+_dvc_metrics_add=''
 _dvc_metrics_add_COMPGEN=_dvc_compgen_files
-_dvc_metrics_diff='--targets -t --type -x --xpath -R --show-json --show-md --no-path --old'
-_dvc_metrics_modify='-t --type -x --xpath'
-_dvc_metrics_modify_COMPGEN=_dvc_compgen_files
+_dvc_metrics_diff='--targets -R --recursive --all --show-json --show-md --no-path --old'
 _dvc_metrics_remove=''
 _dvc_metrics_remove_COMPGEN=_dvc_compgen_files
-_dvc_metrics_show='-t --type -x --xpath -a --all-branches -T --all-tags -R --recursive'
-_dvc_metrics_show_COMPGEN=_dvc_compgen_files
+_dvc_metrics_show='-a --all-branches -T --all-tags --all-commits -R --recursive --show-json'
+_dvc_metrics_show_COMPGEN=_dvc_compgen_DVCFiles
 _dvc_move=''
 _dvc_move_COMPGEN=_dvc_compgen_files
 _dvc_params='diff'
 _dvc_params_diff='--all --show-json --show-md --no-path'
 _dvc_pipeline='list show'
 _dvc_pipeline_list=''
-_dvc_pipeline_show='-c --commands -o --outs --ascii --dot --tree -l --locked'
+_dvc_pipeline_show='-c --commands -o --outs -l --locked --ascii --dot --tree'
 _dvc_pipeline_show_COMPGEN=_dvc_compgen_DVCFiles
-_dvc_plots='show diff'
-_dvc_plots_show='-t --template -o --out -x -y --show-json --no-csv-header --title --xlab --ylab'
-_dvc_plots_diff='-t --template --targets -o --out -x -y --show-json --no-csv-header --title --xlab --ylab'
-_dvc_pull='-j --jobs -r --remote -a --all-branches -T --all-tags -f --force -d --with-deps -R --recursive'
+_dvc_plots='show diff modify'
+_dvc_plots_diff='--targets -t --template -x -y --no-csv-header --title --xlab --ylab -o --out --show-vega'
+_dvc_plots_modify='-t --template -x -y --no-csv-header --title --xlab --ylab --unset'
+_dvc_plots_modify_COMPGEN=_dvc_compgen_DVCFiles #?
```

Comment:
need to check this line

Reply:
It is more complex than that (it wants specific output file/dir that is marked as plot in respective dvcfile), but `files` works fine for now.